tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -114,9 +117,11 @@ class RLoopManager : public RNodeBase {
    const ELoopType fLoopType; ///< The kind of event loop that is going to be run (e.g. on ROOT files, on no files)
    const std::unique_ptr<RDataSource> fDataSource; ///< Owning pointer to a data-source object. Null if no data-source
    std::map<std::string, std::string> fAliasColumnNameMap; ///< ColumnNameAlias-columnName pairs
-   std::vector<TCallback> fCallbacks;                      ///< Registered callbacks
-   std::vector<TOneTimeCallback> fCallbacksOnce; ///< Registered callbacks to invoke just once before running the loop
-   std::vector<Callback_t> fDataBlockCallbacks; ///< Registered callbacks to call at the beginning of each "data block"
+   std::vector<RDFInternal::RCallback> fCallbacks;         ///< Registered callbacks
+   /// Registered callbacks to invoke just once before running the loop
+   std::vector<RDFInternal::ROneTimeCallback> fCallbacksOnce;
+   /// Registered callbacks to call at the beginning of each "data block"
+   std::vector<RDFInternal::Callback_t> fDataBlockCallbacks;
```

Comment:
You're sure this work with doxygen? I thought the docstring had to be in the same line after the member declaration?

Reply:
afaik, line before with `///` or same line with `//<`

Reply:
yep see e.g. https://root.cern/doc/master/classROOT_1_1Detail_1_1RDF_1_1RLoopManager.html#a4a82e371e7cc9c3c9dc4567f0d0bad76

Reply:
Ah yes, just like for the functions. Okay then!